Dockerfile

```diff
@@ -30,7 +43,7 @@ ENV GLIDE_DOWNLOAD_SHA256 0e2be5e863464610ebc420443ccfab15cdfdf1c4ab63b5eb25d121
 
 # Download and install the glide package manager.
 RUN curl -fsSL ${GLIDE_DOWNLOAD_URL} -o glide.tar.gz \
-    && echo "${GLIDE_DOWNLOAD_SHA256} glide.tar.gz" | sha256sum -c - \
+    && echo "${GLIDE_DOWNLOAD_SHA256}  glide.tar.gz" | sha256sum -c - \
```

Comment:
This byte is extra sacred because of how long it took to realize that the sha256 for alpine requires two spaces between the sha and the filename, versus the one space in Ubuntu(?). https://github.com/gliderlabs/docker-alpine/issues/174#issuecomment-222951567

Reply:
WOOOOOW